src/components/Header/Sidebar.js

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { Link } from "react-router-dom"
+import { IoCloseSharp } from "react-icons/io5"
+import styles from "./Header.module.scss"
+
+const Sidebar = ({ open, handleOpen }) => {
+   return (
+      <>
+         <ul
+            className={styles.sidebar}
+            style={
+               open
+                  ? { transform: "translateX(0px)" }
+                  : { transform: "translateX(-250px)" }
+            }
+         >
+            <div className={styles.closeIcon} onClick={() => handleOpen(false)}>
+               <IoCloseSharp />
+            </div>
+            <Link to="/">
+               <li>Home</li>
```

Comment:
put Link under li
the first child to ul/ol should be li as per semantic

Reply:
changed the position of li and Link as suggested above.